internal/lookout/ui/jest.config.js

```diff
@@ -0,0 +1,15 @@
+// Sync object
```

Comment:
https://create-react-app.dev/docs/running-tests/#coverage-reporting

Reading this I don't know if we need a special Jest config for this.  Could you remove and see if you can just add --test?

Reply:
My bad, we don't need the jest.config.js file. I have changed the jest config in package.json to spit out coverage report in cobertura (as required by Codecov).

Reply:
Once you remove this I think it is good to go!

Reply:
Please remove Jest config and the package-lock.json.  We use yarn now so I forgot that shouldn't be there.